commands/channel.go

```diff
@@ -517,10 +518,12 @@ func moveChannelCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("unable to find destination team %q", args[0])
 	}
 
+	var result error
+
 	channels := getChannelsFromChannelArgs(c, args[1:])
 	for i, channel := range channels {
 		if channel == nil {
-			printer.PrintError(fmt.Sprintf("Unable to find channel %q", args[i+1]))
+			result = multierror.Append(result, fmt.Errorf("unable to find channel %q", args[i+1]))
```

Comment:
We should keep printing to the error queue of the printer. Also `args` and `channels` are not correlated, they have different separators.